benchmark/scan.yaml

```diff
@@ -0,0 +1,9 @@
+prelude: |-
+  $LOAD_PATH.unshift(File.expand_path("lib"))
+  require "strscan"
+  scanner = StringScanner.new("test string")
+benchmark:
+  regexp: |
+    scanner.scan(/\w/)
```

Comment:
Should that be `scanner.scan(/test/)` to be a fairer comparison?
In any case, the benchmark results look similar with `/\w/`.

Reply:
It makes sense but `\w` case should be useful for general regular expression usage.
How about adding one more case that uses `/test/`?

Reply:
That makes sense to me, could you add it if you think it makes sense?

Reply:
OK.
Done.